gems/pending/appliance_console/timezone_configuration.rb

```diff
@@ -0,0 +1,95 @@
+require "appliance_console/logging"
+require 'appliance_console/prompts'
+
+module ApplianceConsole
+  class TimezoneConfiguration
+    # Timezone constants
+    TZ_AREAS         = %w(Africa America Asia Atlantic Australia Canada Europe Indian Pacific US).freeze
+    TZ_AREAS_OPTIONS = ["United States", "Canada", "Africa", "America", "Asia", "Atlantic Ocean", "Australia", "Europe",
+                        "Indian Ocean", "Pacific Ocean", CANCEL].freeze
+    TZ_AREAS_MAP     = Hash.new { |_h, k| k }.merge!(
+      "United States"  => "US",
+      "Atlantic Ocean" => "Atlantic",
+      "Pacific Ocean"  => "Pacific",
+      "Indian Ocean"   => "Indian"
+    ).freeze
+    TZ_AREAS_MAP_REV = Hash.new { |_h, k| k }.merge!(TZ_AREAS_MAP.invert).freeze
```

Comment:
Ah nevermind, I see. It's because we are making the default value for any key the key itself except for the ones that are different which are listed in the `merge!`.